src/inference/src/ie_core.cpp

```diff
@@ -121,15 +121,23 @@ void allowNotImplemented(F&& f) {
 
 ov::AnyMap flatten_sub_properties(const std::string& device, const ov::AnyMap& properties) {
     ov::AnyMap result = properties;
-    for (auto&& property : properties) {
-        auto parsed = parseDeviceNameIntoConfig(property.first);
-        if (device.find(parsed._deviceName) != std::string::npos) {
-            if (property.second.is<ov::AnyMap>()) {
-                for (auto&& sub_property : property.second.as<ov::AnyMap>()) {
-                    result[sub_property.first] = sub_property.second;
-                }
-            }
+    if (device.find("AUTO:") != std::string::npos || device.find("MULTI:") != std::string::npos ||
```

Comment:
For compiled_model for device "AUTO", below config list should be flattened before passing to AUTO plugin, in case those configs setting to AUTO Plugin are lost.  So I think the ":" shouldn't be removed. 
{
ov::device::priorities(“CPU”),
ov::device::properties("AUTO",ov::enable_profiling(true), ov::hint::performance_mode(ov::hint::PerformanceMode::THROUGHPUT))},
}